scripts/mainnet-proxy-follower.sh

```diff
@@ -6,13 +6,11 @@ RUNNER=${RUNNER:-cabal new-run --}
 TOPOLOGY=${TOPOLOGY:-"configuration/topology-proxy-follower.json"}
 
 ARGS=(
-        --log-config              "configuration/log-configuration.yaml"
+        --database-path           "./db/"
         --genesis-file            "configuration/mainnet-genesis.json"
-        --genesis-hash            "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb"
-        --node-id                 "0"
         --topology                "${TOPOLOGY}"
-        --port                    "7776"
-        --real-pbft
+        --socket-dir              "./socket/"
```

Comment:
We have the `--socket-dir` and `--database-path` in the common args coming from `lib-node.sh`, why do we need to repeat it here -- and elsewhere?

Reply:
`lib-node.sh` isn't imported into `scripts/mainnet-proxy-follower.sh`